core/src/main/res/values/strings.xml

```diff
@@ -36,52 +36,52 @@
-    <string name="total_time_listened_to_podcasts">Total time of episodes played:</string>
+    <string name="total_time_listened_to_podcasts">Total playtime:</string>
     <string name="statistics_details_dialog">%1$d out of %2$d episodes started.\n\nPlayed %3$s out of %4$s.</string>
-    <string name="statistics_mode">Statistics mode</string>
-    <string name="statistics_mode_normal">Calculate duration that was actually played. Playing twice is counted twice, while marking as played is not counted</string>
+    <string name="statistics_mode">Stats-mode</string>
+    <string name="statistics_mode_normal">Calculate actual playtime duration. Two playthroughs are counted twice, marking as played is not.</string>
     <string name="statistics_mode_count_all">Sum up all episodes marked as played</string>
-    <string name="statistics_speed_not_counted">Notice: Playback speed is never taken into account.</string>
-    <string name="statistics_reset_data">Reset statistics data</string>
-    <string name="statistics_reset_data_msg">This will erase the history of duration played for all episodes. Are you sure you want to proceed?</string>
+    <string name="statistics_speed_not_counted">(Playback speed is never taken into account.)</string>
+    <string name="statistics_reset_data">Reset stats</string>
+    <string name="statistics_reset_data_msg">Erase playback history for all episodes?</string>
     <string name="statistics_counting_since">Since %s,\nyou played</string>
 
     <!-- Download Statistics fragment -->
-    <string name="total_size_downloaded_podcasts">Total size of episodes on the device:</string>
+    <string name="total_size_downloaded_podcasts">Size of episodes on the device:</string>
 
     <!-- Main activity -->
     <string name="drawer_open">Open menu</string>
     <string name="drawer_close">Close menu</string>
     <string name="drawer_preferences">Drawer Preferences</string>
-    <string name="drawer_feed_order_unplayed_episodes">Sort by counter</string>
-    <string name="drawer_feed_order_alphabetical">Sort alphabetically</string>
-    <string name="drawer_feed_order_last_update">Sort by publication date</string>
-    <string name="drawer_feed_order_most_played">Sort by number of played episodes</string>
+    <string name="drawer_feed_order_unplayed_episodes">By counter</string>
+    <string name="drawer_feed_order_alphabetical">A - Z</string>
+    <string name="drawer_feed_order_last_update">Publication date</string>
+    <string name="drawer_feed_order_most_played">Number of played episodes</string>
     <string name="drawer_feed_counter_new_unplayed">Number of new and unplayed episodes</string>
     <string name="drawer_feed_counter_new">Number of new episodes</string>
     <string name="drawer_feed_counter_unplayed">Number of unplayed episodes</string>
     <string name="drawer_feed_counter_downloaded">Number of downloaded episodes</string>
     <string name="drawer_feed_counter_none">None</string>
```

Comment:
A to Z then?
If "None", "Counter" shouldn't be selectable?
Could be that showing a little indication of what it would look like with the user's elements would help?